refinery/core/views.py

```diff
@@ -593,16 +593,6 @@ class DataSetsViewSet(viewsets.ViewSet):
     http_method_names = ['get', 'delete', 'patch']
     lookup_field = 'uuid'
 
-    def get_object(self, uuid):
```

Comment:
This appears to be the correct method to use (https://www.django-rest-framework.org/api-guide/generic-views/#get_objectself). Why change?